druid/single-container-druid/Dockerfile

```diff
@@ -8,24 +6,19 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 ARG DRUID_VERSION
+ARG DRUID_JAR=apache-druid-${DRUID_VERSION}-bin.tar.gz
+ARG DRUID_URL=https://downloads.apache.org/druid/${DRUID_VERSION}/${DRUID_JAR}
+ARG DRUID_SHA=c971ccbb4220fa2ae9f8a0a0f82c3e4b87750d27ac15d863d1e9f415863ccb8b8e5c764122dc8ef76044a42d27e1108b980ca2b88de8aff660db5a22356d6ab7
```

Comment:
In this version the SHA is hardcoded, but previously we were downloading the version and the SHA  and then compering them. Shall we use the previous approach ?

Reply:
Where did you get the `SHA` ?

Reply:
Same URL you were using, there is only one URL in fact

Reply:
But in this case you need to download the SHA every time we change the Druid version from the same sources, right?

Reply:
Yes, that's the point of SHA, otherwise it is literally useless :)

Reply:
If the two files were in two different sources I'm totally with you, but they are in the same source.I was downloading the two files at the same time: 
```
 RUN wget -q -O /tmp/druid.tar.gz ${DRUID_DOWNLOAD_URL} \
    && wget -q -O /tmp/druid.tar.gz.sha512 ${DRUID_SHA_URL}
```
So, I trusted the source one time as you did :)). But mine was two files that is the difference. I just wanted to keep this automatic as much as possible, that was the reason behind my decision. But this one works for me.